server/Labels/messageBus.js

```diff
@@ -0,0 +1,25 @@
+var messageBus = {
```

Comment:
I'm also a little confused by this, is this Tiffany's? I'm just curious how you got it since it hasn't been pull requested yet as far as I can tell :-) Is it ready to be merged into master as it is?